### ISSUE TITLE
executablePath needs to be explicitly set on latest version of pptr

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2992]
If not using chrome then the process crashes

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.54.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.54.0...4.54.1) (2022-12-25)

- Release 4.54.1 [`cf77a9e`](https://github.com/open-wa/wa-automate-nodejs/commit/cf77a9efdfdf696c0101f3b0b2ade94b94e29a3d)
- fix: #2992 [`e19d031`](https://github.com/open-wa/wa-automate-nodejs/commit/e19d031a8dbcd74ff90167135226972f3ab2764a)
- 🥚 patches update [`ca66c90`](https://github.com/open-wa/wa-automate-nodejs/commit/ca66c90bd293ec18e21630678ea8ecd4833655d1)
- updated types-only package [`ce331b2`](https://github.com/open-wa/wa-automate-nodejs/commit/ce331b225c10f2d57f2fcc068359f91f957500a3)
